modules/nw-dns-forward.adoc

```diff
@@ -5,7 +5,16 @@
 [id="nw-dns-forward_{context}"]
 = Using DNS forwarding
 
-You can use DNS forwarding to override the forwarding configuration identified in `/etc/resolv.conf` on a per-zone basis by specifying which name server should be used for a given zone. If the forwarded zone is the Ingress domain managed by {product-title}, then the upstream name server must be authorized for the domain.
+You can use DNS forwarding to change the forwarding configuration settings in the following ways:
+
+* Override forwarding configuration in the `/etc/resolv.conf` file on a per-zone basis by specifying which name server should be used for a given zone. If the forwarded zone is the Ingress domain managed by {product-title}, then the upstream name server must be authorized for the domain.
+* Override the default upstream in the `/etc/resolv.conf` file by providing a list of upstream DNS servers.
+* Override the default forwarding policy.
+
+[NOTE]
+=====
+You can specify both `/etc/resolv.conf` and the upstream resolvers for the default domain.
```

Comment:
Can you be more explicit here?

Specify /etc/resolv.conf ... for the default domain. What does that mean? Specify a default domain in the /etc/resolv.conf file?

Same for this:
Specify the upstream resolvers for the default domain. What does this mean?

Reply:
Comment addressed.